File: leetcode/js/generate-a-string-with-characters-that-have-odd-counts.js
/**
 * @param {number} n
 * @return {string}
 */
var generateTheString = function (n) {
  if (n % 2 === 0) {
    return "a" + "b".repeat(n - 1);
  } else {
    return "a".repeat(n);
  }
};

File: leetcode/rust/generate-a-string-with-characters-that-have-odd-counts.rs
impl Solution {
    pub fn generate_the_string(n: i32) -> String {
        if n % 2 != 0 {
            "a".repeat(n as usize)
        } else {
            format!("a{}", "b".repeat((n - 1) as usize))
        }
    }
}
